package.json

```diff
@@ -14,6 +14,9 @@
     "start": "yarn storybook",
     "test": "jest",
     "test:watch": "jest --watch",
+    "lint": "tslint src/**/*.{ts,tsx} -t verbose",
+    "lint:fix": "tslint --fix src/**/*.{ts,tsx} -t verbose",
```

Comment:
Can be simplified by reusing the `lint` script as yarn passes parameters on

```
"lint:fix": "yarn lint --fix",
```

Reply:
done! 👍